src/components/NumsOperations.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react';
+
+const numsOperations = [
+  { key: 'clear', value: 'AC' },
+  { key: 'plusMinus', value: '+/-' },
+  { key: 'modulo', value: '%' },
+  { key: 'divide', value: '/' },
+  { key: 'seven', value: 7 },
+  { key: 'eight', value: 8 },
+  { key: 'nine', value: 9 },
+  { key: 'multiply', value: '*' },
+  { key: 'four', value: 4 },
+  { key: 'five', value: 5 },
+  { key: 'six', value: 6 },
+  { key: 'subtract', value: '-' },
+  { key: 'one', value: 1 },
+  { key: 'two', value: 2 },
+  { key: 'three', value: 3 },
+  { key: 'plus', value: '+' },
+  { key: 'zero', value: 0 },
+  { key: 'dot', value: '.' },
+  { key: 'equals', value: '=' },
```

Comment:
[OPTIONAL] I think you can use the value as the key for each button in this case, as each value is unique.

Reply:
each key is also unique... I didn't want to use special characters as keys.